test/utils/test_utils.go

```diff
@@ -0,0 +1,46 @@
+package test_utils
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/gruntwork-io/terratest/modules/helm"
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	resultsFolderPath = "./testspec/%v"
```

Comment:
Let's not put %v here. Let is be a proper directoryPath. Wherever required, let's get the absolute file path from this directory and the file name.

Reply:
Done